protocol.md

```diff
@@ -118,6 +118,7 @@ Following are SysEx commands used for core features defined in this version of t
 ```
 EXTENDED_ID                 0x00 // A value of 0x00 indicates the next 2 bytes define the extended ID
 RESERVED               0x01-0x0F // IDs 0x01 - 0x0F are reserved for user defined commands
+REPORT_FEATURES             0x65 // report which optional Firmata features are supported by the firmware
```

Comment:
I could see a need in the future for having separate `REPORT_CORE_FEATURES` and `REPORT_OPTIONAL_FEATURES` where `REPORT_CORE_FEATURES` would not report a version for each feature since they are all covered under the Firmata protocol version. The distinction would be useful for ConfigurableFirmata where it's possible to exclude certain core features from a firmware build.